src/main/resources/operator/aws-eks/aws-eks-cluster.yaml

```diff
@@ -0,0 +1,331 @@
+---
```

Comment:
We define in AwsEksProvider a lot of values.
But seems like all of them will be ignored, if to change anything.
Because this file is used as is.
Instead all the configurable values have to be replaced in this file before actually applying it.

Use `YamlFileUtil.overlayFile(file, pairs)` as we do it for other Yaml files.

Reply:
while creating the cluster, we are passing parameters for configured values.